packages/select/src/select-dropdown.vue

```diff
@@ -29,11 +30,15 @@ export default {
     // computed
     const popperClass = computed(() => select.props.popperClass)
     const isMultiple = computed(() => select.props.multiple)
-    const minWidth = computed(() => select.selectWrapper?.getBoundingClientRect().width + 'px')
+    const minWidth = ref('')
 
     onMounted(() => {
       // TODO: updatePopper
       // popper.value.update()
+
+      setTimeout(() => {
```

Comment:
Instead of using magical `setTimeout` we need to make sure that whenever `getBoudingClientRect()` is called, the ref is actually there. So the way you making it `ref` is correct, but then you need to watch `select.selectWrapper` for it's existence like `watch(() => select.selectWrapper, (val) => minWidht.value = val.getBoundingClientRect() + 'px');

Reply:
setTimeout could not save us from fail every single time.

Reply:
Ok, I use resize-observer instead.